Owncloud iOs Client/InstantUpload/InstantUpload.m

```diff
@@ -0,0 +1,215 @@
+//
+//  InstantUpload.m
+//  Owncloud iOs Client
+//
+//  Created by Jon Schneider on 5/20/16.
+//
+
+#import <CoreLocation/CoreLocation.h>
+#import <Photos/Photos.h>
+
+#import "InstantUpload.h"
+
+#import "AppDelegate.h"
+#import "constants.h"
+
+#import "ManageAppSettingsDB.h"
+#import "ManageUsersDB.h"
+#import "UserDto.h"
+
+#define kPhotoLibraryAccessPermissionDeniedAlert [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"access_photos_library_not_enabled", nil) message:NSLocalizedString(@"message_access_photos_not_enabled", nil) delegate:nil cancelButtonTitle:NSLocalizedString(@"ok", nil) otherButtonTitles:nil]
+
+//TODO: Language should be for background upload, not general Instant Upload
+#define kLocationAccessAlwaysPermissionDeniedAlert [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"location_not_enabled", nil) message:NSLocalizedString(@"message_location_not_enabled", nil) delegate:nil cancelButtonTitle:NSLocalizedString(@"ok", nil) otherButtonTitles:nil]
```

Comment:
I would suggest to instead of duplicate alertview code use a method like https://github.com/owncloud/ios/blob/master/Owncloud%20iOs%20Client/Files/Preview/DetailView/DetailViewController.m#L1512-L1515 and just call it with the localized string like https://github.com/owncloud/ios/blob/master/Owncloud%20iOs%20Client/Files/Preview/DetailView/DetailViewController.m#L1002 
This would make also  more easier the transition that we plan to do it in a near future to the new UIAlertController instead of the deprecated UIAlertView.
We could isoleted also the method in an utils class.